.github/workflows/clean_pr.yml

```diff
@@ -19,4 +19,13 @@ jobs:
             printf 'The following files were both added and deleted in this PR:\n%s\n' "$ad"
             exit 1
           fi
-
+      - name: Check for added-and-modified images
+          git fetch --quiet origin "$GITHUB_BASE_REF"
```

Comment:
You forgot the `run: |`; the workflow didn't run at all.

Reply:
woops.